avocado/core/plugin_interfaces.py

```diff
@@ -424,6 +424,16 @@ async def update_requirement_cache(runtime_task, result):
         :type result: `avocado.core.teststatus.STATUSES`
         """
 
+    @abc.abstractmethod
+    def is_operational(self):
+        """Checks whether this spawner is operationally capable to perform.
+
+        :result: whether or not this spawner is operational on this system,
+                 that is, whether it has all its requirements set up and
+                 should be ready to operate successfully.
```

Comment:
The "ready to operate successfully" also sounds a bit strange to me - "ready to operate", "ready to be used", "ready for use", "ready to operate properly"?

Reply:
Well, one can argue that it's possible to operate but in a broken or misbehaving way.  I think the "operate successfully" adds that extra level of clarity.

Reply:
I generally consider "operational" to always assume no known or expected errors but this was just a small nitpick anyway.